packages/editor/src/components/character-count/index.js

```diff
@@ -0,0 +1,15 @@
+/**
+ * WordPress dependencies
+ */
+import { withSelect } from '@wordpress/data';
+import { count as characterCount } from '@wordpress/wordcount';
+
+function CharacterCount( { content } ) {
+	return characterCount( content, 'characters_including_spaces' );
+}
+
+export default withSelect( ( select ) => {
+	return {
+		content: select( 'core/editor' ).getEditedPostAttribute( 'content' ),
+	};
+} )( CharacterCount );
```

Comment:
If we're using a functional component for `CharacterCount` (which we should :+1: ), I think it'd be better to use the `useSelect` hook rather than the `withSelect` higher-order component, no?

Reply:
Right! We should refactor `WordCount` to functional as well. I'll create an issue for that.